concepts/05 Widgets/DataGrid/00 Getting Started with DataGrid/40 Edit and Validate Data.md

```diff
@@ -0,0 +1,187 @@
+Users can add new records and update or delete existing records. To allow each of these operations, enable the [allowAdding](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#allowAdding), [allowUpdating](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#allowUpdating), and [allowDeleting](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#allowDeleting) options in the [editing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/) object. The **DataGrid** also supports multiple [edit modes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#mode). This tutorial uses the popup edit mode.
+
+DevExtreme includes a validation engine that validates edited values before they are saved. This engine supports different validation rule types, such as [Email](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/EmailRule/), [Compare](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/CompareRule/), [Range](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/RangeRule/), and more. Validation rules are specified per column; one column can have multiple rules. In this tutorial, select columns have the [Required](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/RequiredRule/) rule.
```

Comment:
In this tutorial, select columns have the [Required](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/RequiredRule/) rule.

I don't understand what does 'select columns' means here

Reply:
```suggestion
DevExtreme includes a validation engine that validates edited values before they are saved. This engine supports different validation rule types, such as [Email](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/EmailRule/), [Compare](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/CompareRule/), [Range](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/RangeRule/), and more. Validation rules are specified per column; one column can have multiple rules. In this tutorial, several columns have the [Required](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/RequiredRule/) rule.
```

Reply:
LGTM